LeggedRobotCfg:
  init_state:
    # target angles [rad] when action = 0.0
    default_joint_angle:
      L_HAA_joint: 0.0
      L_HFE_joint: 0.0
      L_KFE_joint: 0.0

      R_HAA_joint: 0.0
      R_HFE_joint: 0.0
      R_KFE_joint: 0.0

  control:
    # PD Drive parameters:
    stiffness: 20.0
    damping: 0.5
    action_scale: 0.5
    decimation: 10
    user_torque_limit: 30

  normalization:
    clip_scales:
      clip_observations: 100.
      clip_actions: 100.
    obs_scales:
      lin_vel: 2.0
      ang_vel: 0.25
      dof_pos: 1.0
      dof_vel: 0.05
      height_measurements: 5.0

  size:
    actions_size: 6
    observations_size: 24
    commands_size: 3
    obs_history_length: 10
    encoder_output_size: 3
    gait_generator_output_size: 8
